feasibility-dsf-process/src/test/java/de/medizininformatik_initiative/feasibility_dsf_process/service/StoreLiveResultTest.java

```diff
@@ -72,6 +77,9 @@ public void testDoExecute_MeasureReportReferenceIsAddedToTask() throws Exception
         when(execution.getVariable(BPMN_EXECUTION_VARIABLE_TASK))
                 .thenReturn(task);
 
+        when(readAccessHelper.addLocal(measureReport)).thenReturn(
+                new ReadAccessHelperImpl().addLocal(measureReport));
+
```

Comment:
Just a minor suggestion:
If we were to use a `Spy` instead of a `Mock`, then the outcome would be equal but without the inline instantiation. This is achievable by telling Mockito to call the "real" method.

However, this may require a separate set up method if you want to keep the Interface but use a specific implementation instance.

Reply:
Thanks for pointing me to the `@Spy` annotation. This is the first time that I use mockito and I didn't know about it.

Reply:
If you need a short introduction on this topic then please feel free to reach out to me.